### ISSUE TITLE
udev warnings with recent systemd release

### ISSUE THREAD

**Issue by floppym** [vmware/open-vm-tools#396]
When booting a system with systemd-243, the following output appears in the journal:
```
Dec 20 12:49:20 cisdev systemd-udevd[158]: Configuration file /lib64/udev/rules.d/99-vmware-scsi-udev.rules is marked executable. Please remove executable permission bits. Proceeding anyway.
Dec 20 12:49:20 cisdev systemd-udevd[158]: /lib64/udev/rules.d/99-vmware-scsi-udev.rules:5 Invalid value "/bin/sh -c 'echo 180 >/sys$DEVPATH/device/timeout'" for RUN (char 27: invalid substitution type), ignoring, but please fix it.
Dec 20 12:49:20 cisdev systemd-udevd[158]: /lib64/udev/rules.d/99-vmware-scsi-udev.rules:6 Invalid value "/bin/sh -c 'echo 180 >/sys$DEVPATH/device/timeout'" for RUN (char 27: invalid substitution type), ignoring, but please fix it.
```
The first warning may be resolved by adding `-m 0644` to the install command in Makefile.am.

The other warnings may be resolve by escaping the dollar-sign with another dollar-sign in the `RUN` parameter:
```
--- a/open-vm-tools/udev/99-vmware-scsi-udev.rules
+++ b/open-vm-tools/udev/99-vmware-scsi-udev.rules
@@ -2,6 +2,6 @@
 #
 # This file is part of open-vm-tools

-ACTION=="add", SUBSYSTEMS=="scsi", ATTRS{vendor}=="VMware*", ATTRS{model}=="Virtual disk*", ENV{DEVTYPE}=="disk", RUN+="/bin/sh -c 'echo 180 >/sys$DEVPATH/device/timeout'"
-ACTION=="add", SUBSYSTEMS=="scsi", ATTRS{vendor}=="VMware*", ATTRS{model}=="VMware Virtual S", ENV{DEVTYPE}=="disk", RUN+="/bin/sh -c 'echo 180 >/sys$DEVPATH/device/timeout'"
+ACTION=="add", SUBSYSTEMS=="scsi", ATTRS{vendor}=="VMware*", ATTRS{model}=="Virtual disk*", ENV{DEVTYPE}=="disk", RUN+="/bin/sh -c 'echo 180 >/sys$$DEVPATH/device/timeout'"
+ACTION=="add", SUBSYSTEMS=="scsi", ATTRS{vendor}=="VMware*", ATTRS{model}=="VMware Virtual S", ENV{DEVTYPE}=="disk", RUN+="/bin/sh -c 'echo 180 >/sys$$DEVPATH/device/timeout'"

```

**Comment by jonathanvmw** [comment]
This was previously reported in #371 and fixed in the devel branch with commit f1dab8d .

**Comment by floppym** [comment]
Sorry about the duplicate report; thanks for the link.
